tests/text/test_blue.py

```diff
@@ -42,6 +44,11 @@
 LIST_OF_REFERENCES = [[REF1A, REF1B, REF1C], [REF2A]]
 HYPOTHESES = [HYP1, HYP2]
```

Comment:
```suggestion
LIST_OF_REFERENCES = ((REF1A, REF1B, REF1C), (REF2A))
HYPOTHESES = (HYP1, HYP2)
```
rather immutable, and so rename the `LIST_` too

Reply:
`(REF2A)` doesn't make it a tuple and maintains same datatype, hence I made it `tuple(REF2A)`